test/unit_tests/js/modules/transition/MoveTransition-spec.js

```diff
@@ -1,4 +1,4 @@
-const BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+const BASE_JS_PATH = '../../../../../cfgov/unprocessed/js/';
```

Comment:
We should probably make this a global node variable that you can just use in the files.

Reply:
Yeah we could probably do something with https://nodejs.org/api/path.html that figures this out.